apps/api-gateway/src/organization/organization.controller.ts

```diff
@@ -318,6 +319,29 @@ export class OrganizationController {
 
     return res.status(HttpStatus.OK).json(finalResponse);
   }
+
+   /**
+  * @returns DID list of organization
+  */
+
+   @Get('/:orgId/dids')
+   @Roles(OrgRoles.OWNER, OrgRoles.ADMIN, OrgRoles.ISSUER)
+   @ApiBearerAuth()
+   @UseGuards(AuthGuard('jwt'), OrgRolesGuard)
+   @ApiResponse({ status: HttpStatus.OK, description: 'Success', type: ApiResponseDto })
+   @ApiOperation({ summary: 'Fetch organization dids', description: 'Get all dids from organization' })
```

Comment:
dids must be `DIDs` in a summary and description
```suggestion
   @ApiOperation({ summary: 'Fetch organization DIDs', description: 'Get all DIDs from organization' })
```